clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy

```diff
@@ -181,8 +186,9 @@ class OperationsController {
     }
   }
 
-  private Map<String, String> start(List<AtomicOperation> atomicOperations) {
-    Task task = orchestrationProcessor.process(atomicOperations)
+  private Map<String, String> start(List<AtomicOperation> atomicOperations, String key) {
+    key = key ?: UUID.randomUUID()
```

Comment:
I'd stick a .toString on the end of that because I don't know what groovy is going to do with the assignment of a UUID object onto the key variable at this point